v4/source/files.yaml

```diff
@@ -5,29 +5,45 @@
       summary: Upload a file
       description: |
         Uploads a file that can later be attached to a post.
+
+        This request can either be a multipart/form-data request with a channel_id, files and optional
```

Comment:
Actually, can you add a note here about the new functionality only being available in 4.8+ servers?